connector-email/src/main/resources/application.properties

```diff
@@ -0,0 +1,52 @@
+notifications.connector.endpoint-cache-max-size=100
+notifications.connector.kafka.incoming.group-id=notifications-connector-email
+notifications.connector.kafka.incoming.topic=${mp.messaging.tocamel.topic}
+notifications.connector.kafka.outgoing.topic=${mp.messaging.fromcamel.topic}
+notifications.connector.name=email_subscription
+notifications.connector.redelivery.counter-name=camel.email.retry.counter
+notifications.connector.redelivery.delay=1000
+notifications.connector.redelivery.max-attempts=2
```

Comment:
There are default values in `ConnectorConfig`.
```suggestion
```

Reply:
Removed!